cypress/component/AllServicesDropdown/AllServicesDropdown.cy.tsx

```diff
@@ -23,11 +23,10 @@ describe('<AllServicesDropdown />', () => {
   it('should close all services dropdown in link matches current pathname', () => {
     function checkMenuClosed() {
       cy.get('.pf-v5-c-menu-toggle__text').click();
-      cy.contains('All services').should('exist');
+      cy.contains('View all').should('exist');
       cy.contains('Favorites').click();
       cy.contains('Test section').click();
       cy.contains('Test link').click();
-      cy.contains('All services').should('not.exist');
```

Comment:
You want to verify the dropdown is closed. YOu are now looking for a different string.
```
cy.contains('View all').should('not.exist');
```